AUTHORS

```diff
@@ -39,3 +39,4 @@ Chris St. Pierre (chris.a.st.pierre@gmail.com)
 Florian Weimer (fw@deneb.enyo.de)
 Adam Young (ayoung@redhat.com)
 Lukas Zapletal (lzap+git@redhat.com)
+Daniel Alley (dalley@redhat.com)
```

Comment:
Woo! But note that the list is alphabetical by last name, so move this up about 40 lines.